flowmachine/flowmachine/core/init.py

```diff
@@ -216,14 +219,10 @@ def _init_logging(log_level, write_log_file):
     except (AttributeError, TypeError):
         log_level = logging.ERROR
     true_log_level = logging.getLevelName(log_level)
-    formatter = logging.Formatter(
```

Comment:
Not a big deal, but what's the rationale for removing this formatter? (In particular, what do messages look like after this change?) It should only be relevant for on-screen formatting, right? Why not keep the structure of those messages as we had them before?

Reply:
Makes the messages easier to parse. Without removing the formatter, you get the structlog message jammed after the stuff the formatter puts in. So, if you want to be able to parse all the logs you'd then need to do special handling to be able to parse any debugging ones. Sans formatter, every log message is a valid json object.

Reply:
👍